Header header
int64 frame_num
uint32 timestamp_t3s_sec
uint32 timestamp_t3s_usec
int8 image_width
int8 image_height
float32[] temperaturedata